apis/opts/diskquota.go

```diff
@@ -9,17 +9,21 @@ import (
 func ParseDiskQuota(quotas []string) (map[string]string, error) {
 	var quotaMaps = make(map[string]string)
 
+	if quotas == nil {
```

Comment:
Actually `quotas` is a slice of string, then I think using `len(quota) == 0` will be better. Since then it will cover two cases:
* first, the slice is a nil;
* second, the slice is not nil, but an empty slice.

WDYT? @hellolijj

Reply:
A very good advise, i totally agree